packages/preview/fontawesome-0.1.0/lib.typ

```diff
@@ -0,0 +1,2491 @@
+/*
+
+typst-fontawesome
+
+https://github.com/duskmoon314/typst-fontawesome
+
+generated by typst-fontawesome helper python script
+
+*/
+
+#let FA_VERSION = state("fa-version", "Font Awesome 6")
```

Comment:
why does this use a state? wrapping everything in locate below can incur significant overhead.

Reply:
My goal is to provide a function to set the whole icon set. Since we cannot use `set rule` on non-element functions, I find out that `state` can achieve the goal. Thus, one can use `FA_SET.update("Brands")` to change the default set to `Brands` and use `fa-icon("github")`, which cannot be directly used in the `Free` set.

If there is a better way to achieve this, I would like to change this ASAP.

Reply:
I see. In that case it should be documented how to use that in the README.

Reply:
Oh, right. I put it in the example of the original repo and forget the README.